app/javascript/utils/formUtils.js

```diff
@@ -40,6 +40,28 @@ const submitErrors = (formApi) => {
   return formApi.submits > 0 ? formApi.errors : mapValues(formApi.errors, () => null)
 }
 
+export const maxLengthMap = {
+  first_name: 40,
+  middle_name: 20,
+  last_name: 40,
+  day: 2,
+  month: 2,
+  year: 4,
```

Comment:
are you using these date validations anywhere?

Reply:
Ah yes, the dates already have validation, removing these.